nltk/stem/snowball.py

```diff
@@ -516,7 +516,7 @@ def __Suffix_Verb_Step1(self, token):
 
     def __Suffix_Verb_Step2a(self, token):
         for suffix in self.__suffix_verb_step2a:
-            if token.endswith(suffix):
+            if token.endswith(suffix) and len(token) > 3:
```

Comment:
Just out of curiosity, is there a linguistic reason to avoid words with 2 characters?

Reply:
We didn't study the case of words that have 2 length yet,  We're mentioned it in the list of our todos too.